### DISCUSSION TITLE
Build confirmation on steamworks

### DISCUSSION THREAD

**Discussion by furai** [dyc3/steamguard-cli#360] [Q&A] [answered]
Hey folks!

I have a question. can this tool also handle publishing build confirmations on steam?

I'm talking about their new requirement from not so long ago [which you can read about more here](https://store.steampowered.com/news/group/4145017/view/3749866608167579205).

Cheers,
Furai

**Comment by dyc3** [comment, marked as answer]
From the article you linked

> We’ll be adding support to use the Steam Mobile App to also approve default builds being set live for released apps or inviting new users on the Steamworks site. The Steam Mobile App will be an alternative to entering an SMS confirmation code.

So it looks like yes, it should work since we use the same APIs as the mobile app. However, steamguard-cli will show the confirmation as "Unknown". If you encounter this, please run it with the `-v debug` arg and file an issue so I can add it to the list of known confirmation types.

**Reply by furai** [reply]
Ok, will do. So far I haven't had the need to do it. Just to confirm, I have to use trade subcommand for that?

**Reply by dyc3** [reply]
Yup
